transcom/mymove
# [MB-16988 and MB-16987] Display SIT Entry on 1st day SIT and Departure Date on SIT destination delivery service items

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16988)
[Jira ticket](https://dp3.atlassian.net/browse/MB-16987)

## Summary

We added SIT entry date and SIT departure date to the details displayed for SIT service items.

This is branched off of https://github.com/transcom/mymove/pull/11355

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

https://github.com/transcom/mymove/assets/2627844/cb029a51-c525-4b21-9942-16793f194159


Create or find a move that has departed SIT (Testharness scenario HHGMoveInSITDeparted should work).
Sign in as a TOO and find this move.
Go to the Move Task Order page
Scroll down to the service items
Find both Domestic destination 1st day SIT and Domestic destination SIT delivery, and ensure that they show the entry date and the departure date, respectively.
Make sure that none of the other SIT service items show the entry or departure dates.

Testharness scenario example showing both entry and departure date examples:

<img width="944" alt="Screenshot 2023-09-12 at 1 26 24 PM" src="https://github.com/transcom/mymove/assets/108361430/4a8e8b56-2308-41e4-91b6-7036679e7562">


### Frontend
